llvm/include/llvm/Analysis/ValueLattice.h

```diff
@@ -492,7 +667,7 @@ class ValueLatticeElement {
   void setNumRangeExtensions(unsigned N) { NumRangeExtensions = N; }
 };
 
-static_assert(sizeof(ValueLatticeElement) <= 40,
+static_assert(sizeof(ValueLatticeElement) <= 80,
```

Comment:
I was wondering why the size increase was *this* large, and apparently the answer is: APFloat is currently throwing 8 bytes out of the window because it inherits from APFloatBase in a situation where empty base class optimization does not apply, because the first member if a union that contains members that also inherit from APFloatBase.